tests/testthat/test-fotmob.R

```diff
@@ -59,7 +59,15 @@ test_that("fotmob_get_league_matches() works", {
   # must also provide country
   expect_error(
     fotmob_get_league_matches(
-      country = "Premier League"
+      league_name = "Premier League"
```

Comment:
this should have been `league_name` before 😅 . it was easy to miss since this call is supposed to fail anyways